site/docs/web-configurator/web-configurator.mdx

```diff
@@ -18,7 +18,7 @@ Select the button labels to be displayed in the usage guide:
 <InputLabelSelector />
 <br />
 
-GP2040-CE contains a built-in web-based configuration application which can be started by holding <Hotkey buttons={["S2"]}/> when plugging your controller into a PC or by holding <Hotkey buttons={["S2"]}/>, <Hotkey buttons={["B3"]}/>, and <Hotkey buttons={["B4"]}/> together for five seconds while the controller is plugged in. Then access <http://192.168.7.1> in a web browser to begin configuration.
+GP2040-CE contains a built-in web-based configuration application which can be started by holding <Hotkey buttons={["S2"]}/> when plugging your controller into a PC or by holding <Hotkey buttons={["S1", "B3", "B4"]}/> for five seconds while the controller is plugged in. Then access <http://192.168.7.1> in a web browser to begin configuration.
```

Comment:
Here is says the webconfig shortcut is S1+B3+B4.

Reply:
Correction has been made. It should be `S2+B3+B4`.